### DISCUSSION TITLE
Error  Conflict: Multiple assets emit different content to the same filename index.html

### DISCUSSION THREAD

**Discussion by AsuraKev** [web-infra-dev/rspack#7993] [Q&A]
Hi there, does anyone came across similar issue like this when enabled the rspack.HtmlRspackPLugin?

![image](https://github.com/user-attachments/assets/e08b6730-9058-4c94-b252-9a4a5a3e5883)

**Comment by greyblake** [comment]
I have the same problem.. Still looking for a solution

**Reply by tommyxst** [reply]
Hi, do you find a solution for this problem? I got the same issue and cannot figure out why :(
